azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -515,6 +517,7 @@ private void ajaxFetchExecFlowLogs(HttpServletRequest req,
     } catch (ExecutorManagerException e) {
       throw new ServletException(e);
     }
+    WebMetrics.INSTANCE.setFetchLogLatency(System.currentTimeMillis() - startMs);
```

Comment:
Have you considered timers? http://metrics.dropwizard.io/3.1.0/getting-started/#timers